src/command/defaults/ExperienceCommand.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\command\defaults;
+
+use pocketmine\command\CommandSender;
+use pocketmine\command\utils\InvalidCommandSyntaxException;
+use pocketmine\entity\Attribute;
+use pocketmine\permission\DefaultPermissionNames;
+use pocketmine\utils\AssumptionFailedError;
+use pocketmine\utils\Limits;
+use pocketmine\utils\TextFormat;
+use function abs;
+use function count;
+use function max;
+use function min;
+use function str_ends_with;
+use function substr;
+
+class ExperienceCommand extends VanillaCommand{
+
+	public function __construct(){
+		parent::__construct(
+			"xp",
+			"Adds or removes player experience",
+			"/xp <experience[L]> [player]"
+		);
+		$this->setPermissions([
+			DefaultPermissionNames::COMMAND_XP_SELF,
+			DefaultPermissionNames::COMMAND_XP_OTHER
+		]);
+	}
+
+	public function execute(CommandSender $sender, string $commandLabel, array $args){
+		if(count($args) < 1){
```

Comment:
```suggestion
		if(count($args) !== 2){
```

Reply:
This is incorrect. “Player” is optional argument and equals to sender by default

Reply:
you're right